h2o-py/tests/testdir_parser/pyunit_parquet_adjust_timezone.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+sys.path.insert(1, "../../../")
+from tests import pyunit_utils
+from h2o.frame import H2OFrame
+import tempfile
+import h2o
+
+'''
+Adjust timestamp parquet
+'''
+
+def adjust_timestamp_parquet():
+    with tempfile.TemporaryDirectory() as dir:
+        timestamp_df = H2OFrame({"timestamp": '2024-06-06 06:06:06'})
+        h2o.export_file(timestamp_df, path=dir, format="parquet", write_checksum=False)
+        imported_df = h2o.import_file(dir)
```

Comment:
I wonder if it would be better to have this option as an export_file parameter..

I see you implement the workaround 

> manipulate the data before saving

What if user want to manipulate data into different timezones? In this case, they need to restart the cloud with the new jvm arguments. 

Am I right or I just overthinking it and its not a probable usecase?

Reply:
And what if user want to adjust the timezone just for the one file, and leave it "as is" for the rest of them? It will also require a restart

Reply:
The main use experience issues that you need to think about the future import of parquet file when you start the h2o. I think its worth to have it as an import (not export) option in the function. WDYT @wendycwong? 

Come to think of it, what happen if you import your file with `sys.ai.h2o.parquet.import.timestamp.adjustment=2`
and export it. IMHO it will export it with +2hours.